tutorials/combine-feature-collections/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Combine FeatureCollections into a new FeatureCollection
+description: This basic tutorial shows how users can combine two FeatureCollections into one. It is targeted at beginners.
+author: sabrinaszeto
+tags: beginner, featurecollection
+date_published: 2019-09-18
+---
+
+[Open In Code Editor](https://code.earthengine.google.com/b4a2877c24cdd175bca2d7252cd733de)
+
+This basic tutorial shows how users can combine two `ee.FeatureCollection`s into a new `ee.FeatureCollection`.
+
+## Create two `ee.FeatureCollection` objects
+
+Let's begin by generating two sets of random points within the boundary of Utah state in USA. First, define the boundary of Utah as a geometry.
+
+```
+var UtahGEOMETRY = ee.Geometry.Polygon([
```

Comment:
Could you also create a file alongside the md with the entire runnable JavaScript source? (e.g., example.js)

Reply:
Will work on this as well.